opentelemetry-api/opentelemetry/trace/__init__.py

```diff
@@ -231,3 +232,37 @@ class TraceOptions(int):
 # TODO
 class TraceState(typing.Dict[str, str]):
     pass
+
+
+_TRACER = None
+
+def tracer() -> Tracer:
+    """Gets the current global :class:`~.Tracer` object.
+    If there isn't one set yet, a default will be loaded."""
+
+    global _TRACER #pylint:disable=global-statement
+
+    if _TRACER is None:
+        #pylint:disable=protected-access
+        _TRACER = loader._load_impl(Tracer)
+
+    return _TRACER
+
+def set_preferred_tracer_implementation(
+        factory: typing.Callable[[typing.Type[Tracer]], Tracer]) -> None:
+    """Sets a callback which to query for the tracer implementation.
+
+    See :mod:`opentelemetry.loader` for details.
+
+    This function may not be called after a tracer is already loaded.
+
+    Args:
+            factory: A function that, when called with the :class:`Tracer` type
```

Comment:
Why two levels of indentation here? Also I think it's fine to omit the types in this line since they're in the signature.

Reply:
I fixed the indentation. I also made the description more terse, is that what you meant?

Reply:
Yup, looks good.